### ISSUE TITLE
Add Solana support

### ISSUE THREAD

**Issue by fadeev** [zeta-chain/universalkit#22]

**Comment by lukema95** [comment]
@fadeev I think the first part of the above task (adding a Solana-enabled browser extension wallet) needs to be completed before we can complete [Make Solana deposit function callable from ZetaChainClient and from browser](https://github.com/zeta-chain/toolkit/issues/182). Since RainbowKit only supports EVM wallets, and Phantom is actually a multi-chain wallet (EVM, Solana, and Bitcoin), integrating RainbowKit with Phantom only supports EVM networks. We need Solana's official wallet suite for support, which includes `@solana/ wallet-Adapter-React` and `@solana/ wallet-Adapter-React-UI`, here are the relevant [docs](https://solana.com/developers/courses/intro-to-solana/interact-with-wallets#solana-wallet-adapter-react-ui). So I created a new issue #27
